sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/DicomIO.java

```diff
@@ -51,8 +64,33 @@
  * readMetadataResult.getReadResponse() PCollection<String> failRead =
  * readMetadataResult.getFailedReads() }
  */
+
+// @SuppressWarnings({
+// "nullness" // TODO(https://github.com/apache/beam/issues/20497)
+// })
 public class DicomIO {
 
+  static final String BASE_METRIC_PREFIX = "dicomio/";
+  private static final String LRO_COUNTER_KEY = "counter";
+  private static final String LRO_SUCCESS_KEY = "success";
+  private static final String LRO_FAILURE_KEY = "failure";
+  private static final Logger LOG = LoggerFactory.getLogger(DicomIO.class);
+
+  public static Deidentify deidentify(
+      String sourceDicomStore, String destinationDicomStore, DeidentifyConfig deidConfig) {
+    return new Deidentify(
+        StaticValueProvider.of(sourceDicomStore),
+        StaticValueProvider.of(destinationDicomStore),
+        StaticValueProvider.of(deidConfig));
+  }
+
+  public static Deidentify deidentify(
+      ValueProvider<String> sourceDicomStore,
+      ValueProvider<String> destinationDicomStore,
+      ValueProvider<DeidentifyConfig> deidConfig) {
```

Comment:
DeidentifyConfig is a protobuf generated message. I'm not sure how a ValueProvider of this (not String or Json) type could be used by downstream service, e.g. DataflowTemplate. Probably good to confirm with CC: @bvolpato

In general, we would like the exposed method for user faced XXIO class to be stable in signature